crates/holochain_types/src/app/app_manifest/app_manifest_validated.rs

```diff
@@ -82,10 +73,13 @@ pub enum AppRoleManifestValidated {
         modifiers: DnaModifiersOpt,
         installed_hash: DnaHashB64,
     },
-    /// Disallow provisioning altogether. In this case, we expect
-    /// `clone_limit > 0`: otherwise, no cells will ever be created.
-    Disabled {
-        installed_hash: DnaHashB64,
+    /// Install or locate the DNA, but never create a Cell for this DNA.
+    /// Only allow clones to be created from the DNA specified.
+    /// This case requires `clone_limit > 0`, otherwise no Cells will ever be created.
+    CloneOnly {
         clone_limit: u32,
+        location: DnaLocation,
+        modifiers: DnaModifiersOpt,
```

Comment:
I take it there'll not be a cell filling this spot, and clone cells will be created at runtime. That means that each CreateCloneCell call will provide DNA modifiers. What are the modifiers here for?

Reply:
@jost-s interesting, and, nice find. I had to ask myself if these play any role. And the answer is yes, they do, because [modifiers get overridden one by one](https://github.com/holochain/holochain/blob/ac8cf4161e276312926fe45ceb53e7da7b064fae/crates/holochain_zome_types/src/dna_def.rs#L62-L72), not all together, so even though the `create_clone` call requires some modifiers set, other modifiers may still apply. For instance if the clone cell sets the network seed but leaves the properties field `None`, the properties set here in the manifest will be used.

Reply:
ah, cool, got it